.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,22 @@
+name: Lint Pull Requests
+
+on: 
+  pull_request:
+    branches:
+    - '*'
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: pip install flake8 lintly
+    - name: Lint with flake8
+      run: flake8 --config=tox.ini . | lintly
```

Comment:
Lintly will have to be removed since it doesn't support 3.3. That or add different commands for 3.3 and 3.8.

Reply:
Those scripts run online. Is it really required them to run on 3.3? I am not aware of linting rules differ between 3.3 and 3.8.

Reply:
Yes, Python 3.8 has all kinds of syntax that doesn't exist on 3.3.

Reply:
If you only lint for Python 3.8, we effectively are EOLing ST3.

Reply:
Just someone adding an F string will break 3.3.

Reply:
That's a good point.